File: src/main/java/com/lessmarkup/interfaces/recordmodel/SelectValueModel.java
package com.lessmarkup.interfaces.recordmodel;

public class SelectValueModel {
    private final String text;
    private final String value;
    
    public SelectValueModel(String text, String value) {
        this.text = text;
        this.value = value;
    }
    
    public String getText() {
        return this.text;
    }
    
    public String getValue() {
        return this.value;
    }
}

File: src/main/scala/com/lessmarkup/interfaces/recordmodel/SelectValueModel.scala
/*
 * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
 * If a copy of the MPL was not distributed with this file, You can obtain one at
 * http://mozilla.org/MPL/2.0/.
 */

package com.lessmarkup.interfaces.recordmodel

class SelectValueModel(text: String, value: String) {

  def getText: String = this.text
  def getValue: String = this.value
}